Dockerfile

```diff
@@ -2,13 +2,20 @@ FROM quay.io/centos/centos:stream8
 
 RUN dnf -y module install python39 && dnf -y install python39 python39-pip
 RUN mkdir /app
-ADD https://raw.githubusercontent.com/arcalot/arcaflow-plugin-template-python/main/LICENSE /app
-ADD example_plugin.py /app
-ADD test_example_plugin.py /app
-ADD requirements.txt /app
+RUN chmod 777 /app
```

Comment:
Is the chmod 777 required?

Reply:
Usually you need read & execute on a directory for the owner only.  Do plugins modify the `/app` directory?  If so, then `chmod 700 /app` is sufficient.

Reply:
Consider using something like:

```
RUN mkdir /app \
 && chmod 700 /app
```

See https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run.

Reply:
will do thank you